README.md

```diff
@@ -13,6 +13,7 @@ Sets output variable 'manifestsBundle' which contains the location of the manife
    with:
       renderEngine: 'helm'
       helmChart: './aks-helloworld/'
+      arguments: './aks-helloworld/values-argument.yaml'
```

Comment:
give a better example. This would run a command that doesn't make any sense.

https://helm.sh/docs/helm/helm_template/